EXILED/Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -539,6 +539,21 @@ public virtual void AddRole(Player player)
                     }
                 });
 
+            if (Ammo.Count > 0)
+            {
+                Timing.CallDelayed(
+                    0.5f,
+                    () =>
+                    {
+                        Log.Debug($"{Name}: Adding Ammo to {player.Nickname} inventory.");
+                        foreach (AmmoType type in EnumUtils<AmmoType>.Values)
+                        {
+                            if (type != AmmoType.None)
+                                player.SetAmmo(type, Ammo.ContainsKey(type) ? Ammo[type] == ushort.MaxValue ? InventoryLimits.GetAmmoLimit(type.GetItemType(), player.ReferenceHub) : Ammo[type] : (ushort)0);
+                        }
+                    });
+            }
```

Comment:
cant you move that ammogive logic to a upper delay for items? Or does it lead to errors?

Reply:
I thought it would cause problems but I tested it and it worked so it is now moved.